datavault-webapp/src/main/webapp/WEB-INF/freemarker/admin/roles/isadmin/index.ftl

```diff
@@ -0,0 +1,212 @@
+<#import "*/layout/defaultlayout.ftl" as layout>
+<#-- Specify which navbar element should be flagged as active -->
+<#global nav="admin">
+<@layout.vaultLayout>
+    <#import "/spring.ftl" as spring />
+
+    <div class="container container_layout">
+    <div>
+        <div class="row">
+            <div class="col-sm-4">
+                <h1 class="layout_roles_title">IS Admin Management</h1>
+            </div>
+            <div class="col-sm-8">
+                <!-- todo buttons (or not?) -->
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-sm-4">
+                <a class="layout_misc__subpage_link" href="${springMacroRequestContext.getContextPath()}/admin/roles">Back
+                    to role management</a>
+            </div>
+        </div>
+
+        <div class="layout_misc__add_new">
+            <a id="addNewAdminBtn" href="#" data-toggle="modal" data-target="#add-new-dialog">+Add New User</a>
+        </div>
+
+        <table class="table_layout">
+            <thead>
+            <tr class="tr">
+                <th class="table_layout__title">Name</th>
+                <th class="table_layout__actions">Actions</th>
+            </tr>
+            </thead>
+            <tbody>
+            <#list users as user>
+                <tr class="tr">
+                    <td>${user.getFirstname()} ${user.getLastname()}</td>
+                    <td>
+                        <#if users?size gt 1>
+                            <a href="#" class="btn btn-link modal_layout_action_btn" data-toggle="modal"
+                               data-target="#delete-dialog" data-role-name="${role.name}" data-user-id="${user.getID()}"
+                               data-user-name="${user.getFirstname()} ${user.getLastname()}" role="button"
+                               title="Remove user ${user.getFirstname()} ${user.getLastname()} from the ${role.name} role.">
+                                <i class="fa fa-trash-o"
+                                   style="font-size:24px;color:red;background:MistyRose;border:1px solid black;padding:2px"></i>
+                            </a>
+                        <#else>
+                            <a href="#" class="btn btn-link modal_layout_action_btn" disabled="disabled" role="button"
+                               title="This user is the last IS Admin and cannot be removed.">
```

Comment:
If there's an existing language bundle file (i.e. `messages.properties`) it'd be good to stick these strings in there.

Reply:
It's all hardcoded... I would make things worse by just having a single view use `properties`